### DISCUSSION TITLE
How can I get a new Google idToken?

### DISCUSSION THREAD

**Discussion by AhmetFUsta** [abacritt/angularx-social-login#399] [Q&A] [answered]
We use Google's idTokens to pass to our backend server and confirm the token along with the user's identity. There's an expiration time on these tokens for about an hour, how can we get a new one?

I expected refreshAuthToken to do this but that returns a void wrapped in a promise.

**Comment by jaibatrik** [comment, marked as answer]
I think the short-lived token can be exchanged with a long-lived one in the server. Please check Google's documentation.
